src/engine.rs

```diff
@@ -3,12 +3,17 @@ use std::{
     path::PathBuf,
     time::{Duration, Instant},
 };
-use wasmtime::{Engine, Linker, Module, Store};
+use wasmtime::{Config, Engine, Linker, Module, Store};
 
 use crate::function_run_result::FunctionRunResult;
 
 pub fn run(function_path: PathBuf, input_path: PathBuf) -> Result<FunctionRunResult> {
-    let engine = Engine::default();
+    let engine = if cfg!(target_arch = "x86_64") {
+        // enabling this on non-x86 architectures currently causes an error (as of wasmtime 0.37.0)
+        Engine::new(Config::new().debug_info(true))?
```

Comment:
Does using this flag affect the performance profile of `wasmtime`? Is this something we want to run when the user is trying to benchmark their script? 

If this does affect the performance profile, then we should probably gate this behind a flag that can be used when the user wants to debug.

Reply:
> Does using this flag affect the performance profile of `wasmtime`?

It _shouldn't_ affect what we're measuring as the function runtime. The DWARF symbol transformation _should_ occur as part of module compilation which _should_ happen in the https://docs.rs/wasmtime/0.37.0/wasmtime/struct.Module.html#method.from_file invocation. Outside of compilation, the presence or absence of the symbols should not impact performance.

I tried to validate this experimentally. The spread of runtimes I'm getting for Wasm files with debug symbols and without debug symbols and with symbol generation enabled versus disabled is all over the place. The slowest runs were with debug symbol generation disabled but some of the fastest were also with debug symbol generation disabled. I think that indicates that enabling it doesn't affect the number we're reporting as the runtime materially.

That said, looking the code, I'm not sure we're calculating that runtime number correctly (at least in a statistical sense and possibly in a functional sense) so I'm not confident in my experiment results. I can file a separate issue to track improvements to how to calculate that runtime number.

Reply:
Thanks for looking into it! That makes sense to me. I think we can ship this as-is and look into improving how we measure performance later then.